test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -455,6 +455,7 @@ TEST(LiteInterpreterTest, BuiltinFunction) {
   AT_ASSERT(str == expected);
 }
 
+#if !defined FB_XPLAT_BUILD
```

Comment:
It will be good to add some comments about why this flag is needed.

Reply:
@cccclai Thanks! Updated with the commentary in the caffe2/BUCK file since it's FB specific.